### PR TITLE
Amp 1171 applause to avalon

### PR DESCRIPTION
**Pull request by dfische3**
Converts applause detection segmentation to Avalon XML

### CODE REVIEW

File: tools/amp_segment/applause_detection_to_avalon_xml.xml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<tool id="applause_detection_to_avalon_xml" name="Applause Detection To Avalon XML" version="1.0.0">
+  <description>Converts AMP Applause Detection Segmentation to Avalon Structural Metadata XML</description>
+  <requirements>
+	<requirement type="package" version="3.8">python</requirement>
+  </requirements>
+  <command detect_errors="exit_code">
+  	'$__tool_directory__/applause_detection_to_avalon_xml.py' '$input_segments' '$context_json' '$avalon_xml'
+  </command>
+  <inputs>
+    <param name="input_segments" type="data" format="segment" label="Input Applause Segments" help="Input applause detection segmentation"/>
```
Comment (yingfeng-iu) [inline tools/amp_segment/applause_detection_to_avalon_xml.xml:10]: Let's call it amp_applause_segments, consistent with the applause MGM.

File: tools/amp_segment/applause_detection_to_avalon_xml.xml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    <param name="context_json" type="text" label="HMGM Task Context" help="JSON string containing context information needed for creating a MGM task" >
+    	<sanitizer sanitize="false">
+    	</sanitizer>
+	</param>
+  </inputs>
+  <outputs>
+    <data name="avalon_xml" format="xml" label="Avalon structural metadata XML"/>
```
Comment (yingfeng-iu) [inline tools/amp_segment/applause_detection_to_avalon_xml.xml:17]: I'm asking Maria what would be a good output name. I think we should include something to indicate it's for the structural metadata, but it's such a long name!

**Review by yingfeng-iu — APPROVED** [review]
Could you also make a PR to the master_21.01 branch? upon upgrade, we will switch to that branch

File: tools/amp_segment/applause_detection_to_avalon_xml.xml (hunk 1 above)
Comment (dfische3) [reply tools/amp_segment/applause_detection_to_avalon_xml.xml]: Updated to amp_applause_semgents

File: tools/amp_segment/applause_detection_to_avalon_xml.xml (hunk 2 above)
Comment (dfische3) [reply tools/amp_segment/applause_detection_to_avalon_xml.xml]: Updated!

**Pull request merged**